src/common/dao/group/usergroup.go

```diff
@@ -84,6 +86,27 @@ func GetUserGroup(id int) (*models.UserGroup, error) {
 	return nil, nil
 }
 
+// GetGroupIDByGroupName ...
```

Comment:
Sorry I forgot to mention this in previous review.
By viewing the code it is possible you input 10 group names but get 8 group IDs, and this may be acceptable for current logic, but we should clearly document it in the comment, thoughts?

Reply:
Add